### PR TITLE
chore: automate chrome web store upload

### PR DESCRIPTION
**Pull request by rofe**
Fix: #2 

Moved here from https://github.com/adobe/helix-sidekick/pull/230

### CODE REVIEW

File: build/publish.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (trieloff) [inline build/publish.js:2]: 2022

File: build/updateVersion.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
```
Comment (trieloff) [inline build/updateVersion.js:2]: 2022

**Review by trieloff — APPROVED** [review]
Copyright dates on new files are off

**Review by kptdobe — APPROVED** [review]

**Pull request merged**